app/account/views.py

```diff
@@ -9,7 +9,7 @@
 from ..decorators import merchant_or_vendor_required
```

Comment:
Why is this in the general `/account` folder? I believe that csv uploading should be a vendor restricted action. move to `app/vendor/views.py`